job_spec/WATER_MAP.yml

```diff
@@ -175,3 +181,6 @@ WATER_MAP:
       timeout: 86400
       vcpu: 1
       memory: 126000
+      secrets:
+        - EARTHDATA_USERNAME
+        - EARTHDATA_PASSWORD
```

Comment:
:thinking: I'm not actually sure if `asf_tools` needs EDL credentials... will need to check.

Reply:
It does not.